### DISCUSSION TITLE
May Newsletter

### DISCUSSION THREAD

**Discussion by briancl2** [briancl2/CustomerNewsletter#2] [Customer Newsletter]
This is a personally curated newsletter for just my customers. The purpose is to pull together and share current resources, announcements, and events. I welcome your feedback, and please let me know if you would like to add or remove anyone from this list.

Highlights for this month are Metrics API public beta, Copilot Workspace announcement, in-person [GitHub Galaxy](https://galaxy.github.com/) worldwide series, and Early bird tickets on sale for [GitHub Universe](https://githubuniverse.com/) in October. I will be attending the Chicago Galaxy event on June 13 in-person for anyone in the area that can join. Please register soon because there are limited spots available.

### Measuring the impact of GitHub Copilot - Metrics API public beta
* On April 23, GitHub opened the Copilot Metrics API public beta. This API gives customers direct access to their own usage data for the first time. See more at the [launch blog post](https://github.blog/changelog/2024-04-23-github-copilot-metrics-api-now-available-in-public-beta/) and [May 9 update blog post](https://github.blog/changelog/2024-05-09-github-copilot-metrics-updates/).
* Official GitHub Metrics [Learning Pathway](https://resources-hub.githubapp.com/learn/pathways/copilot/essentials/measuring-the-impact-of-github-copilot/)
* [API Docs](https://docs.github.com/en/rest/copilot/copilot-usage?apiVersion=2022-11-28)
* [Public Repo Visualizer](https://github.com/github-copilot-resources/copilot-metrics-viewer)
* [Public Repo Usage Dashboard](https://github.com/octodemo/Copilot-Usage-Dashboard)
* [Public Repo Developer Metrics](https://github.com/ayodejiayodele/github-developer-metrics)
* [Copilot Usage Action](https://github.com/marketplace/actions/copilot-usage-action)
* [Copilot Metrics API Export Action](https://github.com/marketplace/actions/github-copilot-usage-metrics-api-export)

### Announcing GitHub Copilot Workspace Technical Preview
* On April 29th, GitHub announced the technical preview for our next innovation in AI-driven software development. Unlike Copilot Business and Enterprise which integrate AI features directly into the developer IDE and github.com, Copilot Workspace is a new AI-native developer workflow. Register for the waitlist at the [blog post](https://github.blog/2024-04-29-github-copilot-workspace/)
* Check out this [in-depth product walkthrough video](https://www.youtube.com/watch?v=N64ozm3x88k) from a GitHub partner Rob Bos
* [Independent review](https://every.to/chain-of-thought/i-spent-24-hours-with-github-copilot-workspaces?utm_source=tldrai) of Copilot Workspace by Dan Shipper

### Other GitHub Copilot resources:
* **Visualizing your GitHub metered bill within the Azure portal**: Customers often ask us how they can view their GitHub consumption bill in the Azure Portal. In this brief [video](https://www.youtube.com/watch?v=MgPS6IgcYjc), we'll demonstrate how you can see how much you are spending on each individual GitHub capability like Actions and Copilot as well as set up alerts and budgets to help you manage your costs and avoid surprises.
* Brian’s personally curated Copilot Training & Tips [Youtube playlist](https://www.youtube.com/playlist?list=PLCiDM8_DsPQ1WJ5Ss3e0Lsw8EaijUL_6D), updated monthly
 
### Recent Webinar recordings:
* April 24: [GitHub’s AI + Security Story](https://www.youtube.com/watch?v=8_RGAcz7A-0)
* May 8: [Integrating Security into CI/CD Pipelines on GitHub & Azure DevOps](https://www.youtube.com/watch?v=H6x8l86sqiQ)

### Upcoming Webinars:
* May 21-23: [Microsoft Build Digital Sessions](https://register.build.microsoft.com/): [GitHub session catalog](https://build.microsoft.com/en-US/sessions?search=github&sortBy=relevance)
* May 21: [GitHub Copilot Legal Webinar](https://resources.github.com/amer-github-copilot-legal-webinar-may21/)
* May 22: [GitHub Advanced Security: Compliance and Policy Enforcement](https://www.youtube.com/watch?v=Rl7bhyT8QWI)
* June 5: [Investigating Code Security with GitHub Copilot](https://www.youtube.com/watch?v=StnHNZuuih8)

### Upcoming In-person events
* Bellevue - May 15 - [GitHub Galaxy](https://gh.io/galaxybellevue)
* NYC - May 17 - [Accelerate Application Security with GitHub AI](https://developer.microsoft.com/en-us/reactor/events/22463/)
* Washington, DC - May 21 - [GitHub Galaxy](https://gh.io/galaxydc)
* Seattle - May 21-23 - [Microsoft Build](https://register.build.microsoft.com/): [GitHub session catalog](https://build.microsoft.com/en-US/sessions?search=github&sortBy=relevance)
* San Francisco - June 5 - [GitHub Galaxy](https://gh.io/galaxysanfrancisco)
* Austin - June 6 - [GitHub Galaxy](https://gh.io/galaxyaustin)
* Bengaluru - June 11 - [GitHub Galaxy](https://gh.io/GalaxyBengaluru)
* Mexico City - June 12 - [GitHub Galaxy](https://gh.io/galaxymexicocity)
* Amsterdam - June 12 - [GitHub Galaxy](https://github.registration.goldcast.io/events/12a520d1-e192-4f88-a6ad-9433037a45b3)
* Chicago - June 13 - [GitHub Galaxy](https://gh.io/galaxychicago)
* Toronto - June 17 - [GitHub Galaxy](https://gh.io/galaxytoronto)
* San Francisco - Oct 29-30 - [Early Bird Tickets](https://githubuniverse.com/#tickets) available through July 8

**Comment by alwell-kevin** [comment]
Love this, so well done.
